DOCUMENTATION.md

```diff
@@ -231,6 +231,40 @@ If the request is not successful, the client returns a `NotificationClientExcept
 |`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (LIMIT NUMBER) for today"`<br>`}]`|Refer to [service limits](#service-limits) for the limit number|
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification|
```

Comment:
There are additional errors in the python docs related to document uploads (like "unsupported document type" and "document failed virus scan") that should probably be added to this list

Reply:
Good point - done